### PR TITLE
Release 22.3.1

### PR DESCRIPTION
**Pull request by holmanb**
```
Release 22.3.1
      
Bump the version in cloudinit/version.py to 22.3.1 and
update ChangeLog.
```

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+22.3.1
+ - Identify Huawei Cloud as OpenStack (#1689) [huang xinjie]
```
Comment (blackboxsw) [inline ChangeLog:2]: We don't want to include any commits beyond `Fix v2 interface matching when no MAC (LP: #1986551)` as adding content like Identify Huawei cloud is functionally different than what we released to Ubuntu for 22.3.1

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+ - doc: add reporting suggestion to FAQ (SC-1236) (#1698)
```
Comment (blackboxsw) [inline ChangeLog:3]: We don't want to include any commits beyond `Fix v2 interface matching when no MAC (LP: #1986551)` as adding content like Identify Huawei cloud is functionally different than what we released to Ubuntu for 22.3.1


```suggestion
```

Comment (holmanb) [reply ChangeLog]: > Your 22.3.1 tag is probably pointing at the wrong commitish if this is the Changelog that was constructed

Good catch, thanks. I think I have both tag and changelog corrected now.

Comment (holmanb) [reply ChangeLog]: Here's the updated tag too:
```
git show 22.3.1
  tag 22.3.1
  Tagger: Brett Holman <brett.holman@canonical.com>
  Date:   Thu Sep 1 16:43:48 2022 -0600
  
  Release 22.3.1
  -----BEGIN PGP SIGNATURE-----
  
  iQIzBAABCgAdFiEEOj7zTf3ts7fz/fYD+D93EppevYUFAmMRNbgACgkQ+D93Eppe
  vYXymQ/9GVrEt6rltAmGr6eJ3yS/w+re3cB3++aoKwN9jshAK+YVGg+YmfMo18UD
  kCmklj505i2xCoDGs74LpaMo8t4hZf5/ZyzCEu35m7FCy+u0s9evVjAkWT9f4pkv
  3wTTQ01/GeQUUGwIrFTEs0l38WVhA2+kU4QsBemTo7TPaGcwhbjxDScokMx5MCic
  1SHVHJHIVvwjJHeLIyE5aFvVfDDSYuhbmguqf2xesY6VviIvNB/B8r5JqTHZ/Kel
  amINynffhnN/IaBm+TGmEpQ7MWAIapPbSe0e3WjnU1wOvX0fSUpP9p3aE/FnRiaB
  GUeBexXNOizZDF973a/V2va2ZPSeMiv10tMS8/aLRxFY0gs5bbOttOi88whqG/0R
  oNtSHhKIZ7E2wsEO3L1UkzY99nL/EgiXJZXHbqBf+6k0TrXNWHgGKhTZQKCMa9x+
  JSd3lsIjfn8CXXEICHT4wnnhlnnsc2s9iGinKqFOtjDcsMwkXFLqELrl1fB5NGgH
  H2ocKFyyfFmMjeR3Fq7el22CeOaBEDn9sfUrA36wsmrVoQMGxVl6R8ahr/rAI/4i
  KjfRPYyWxmoNY5HxrPlR9RMiLR+rmfc9eNifHfnGz5E56RnVTp5DRCkgSCqXHHXB
  y5Ah2m7BQXWHzLi++NoBsv5ZGl2AKM7tkMEWtDzKLIi7FT/Ucu8=
  =fZLb
  -----END PGP SIGNATURE-----
  
  commit 1f1dd5f2cc8912617dfef3c7c8aa45ca21534736 (HEAD -> upstream/22.3.1, tag: 22.3.1, origin/upstream/22.3.1)
  Author: Brett Holman <brett.holman@canonical.com>
  Date:   Thu Sep 1 16:06:20 2022 -0600
  
      Release 22.3.1
      
      Bump the version in cloudinit/version.py to 22.3.1 and
      update ChangeLog.
      
      LP: #1986703
  
  diff --git a/ChangeLog b/ChangeLog
  index 0ff766170..26a340e5a 100644
  --- a/ChangeLog
  +++ b/ChangeLog
  @@ -1,3 +1,22 @@
  +22.3.1
  + - Fix v2 interface matching when no MAC (LP: #1986551)
  + - test: reduce number of network dependencies in flaky test (#1702)
  + - docs: publish cc_ubuntu_autoinstall docs to rtd (#1696)
  + - net: Fix EphemeraIPNetwork (#1697) [Alberto Contreras]
  + - test: make ansible test work across older versions (#1691)
  + - Networkd multi-address support/fix (#1685) [Teodor Garzdin]
  + - make: drop broken targets (#1688)
  + - net: Passthough v2 netconfigs in netplan systems (#1650)
  +   [Alberto Contreras] (LP: #1978543)
  + - NM ipv6 connection does not work on Azure and Openstack (#1616)
  +   [Emanuele Giuseppe Esposito]
  + - Fix check_format_tip (#1679) [Alberto Contreras]
  + - DataSourceVMware: fix var use before init (#1674)
  +   [Andrew Kutz] (LP: #1987005)
  + - rpm/copr: ensure RPM represents new clean.d dir artifacts (#1680)
  + - test: avoid centos leaked check of /etc/yum.repos.d/epel-testing.repo
  +   (#1676)
  +
   22.3
    - sources: obj.pkl cache should be written anyime get_data is run (#1669)
    - schema: drop release number from version file (#1664)
  diff --git a/cloudinit/version.py b/cloudinit/version.py
  index ec2621f75..83df251ec 100644
  --- a/cloudinit/version.py
  +++ b/cloudinit/version.py
  @@ -4,7 +4,7 @@
   #
   # This file is part of cloud-init. See LICENSE file for license information.
   
  -__VERSION__ = "22.3"
  +__VERSION__ = "22.3.1"
   _PACKAGED_VERSION = "@@PACKAGED_VERSION@@"
   
   FEATURES = [
```

**Review by blackboxsw — APPROVED** [review]
+1 on the changeset here. We confirmed that we will continue preserving our upstream point releases 22.3.X only as signed/annotated tags which are not expected to be visible directly within the upstream/main history as they represent a point in time release with cherry-pick commits which aligns with our typical distro-based or downstream-based branching strategies.

**Comment by blackboxsw** [comment]
Closing as we don't intend to merge his PR into main.

**Pull request closed without merging**